laser_position:
    ros__parameters:
        DT: 0.001
        START_POINT: [7.5,4.0]
        LASER_ALLOWED_NOISE: 2.0
        process_noise_std: [0.1,0.1,0.05]
        measurement_noise_std: [0.5,0.5,0.1]
        FIELD_SIZE: [15,8]
        FREQUENCY: 0.2